src/v3/plugins/github/relationalView.js

```diff
@@ -127,6 +127,36 @@ export class RelationalView {
     yield* this.comments();
   }
 
+  *parentEntities(): Iterator<ParentEntity> {
+    yield* this.repos();
+    yield* this.issues();
+    yield* this.pulls();
+    yield* this.reviews();
+  }
+
+  *childEntities(): Iterator<ChildEntity> {
+    yield* this.issues();
+    yield* this.pulls();
+    yield* this.reviews();
+    yield* this.comments();
+  }
+
+  *authoredEntities(): Iterator<AuthoredEntity> {
+    yield* this.issues();
+    yield* this.pulls();
+    yield* this.reviews();
+    yield* this.comments();
+  }
+
+  *entities(): Iterator<Entity<Entry>> {
```

Comment:
Yep. If you want to make `Entity` an implementation detail and expose the union only, that's okay with me. (I see now why it makes sense in parallel with the other concepts.)